news/7294.feature.rst

```diff
@@ -0,0 +1,2 @@
+Build environment dependencies are no longer compiled to bytecode during
+installation for a modest but noticeable performance gain.
```

Comment:
Can this be reworded to remove "modest" and "noticable", to something like "for improved performance".

I feel modest and noticable are too subjective and easy to interpret wrong here (I'm not sure I would _notice_ 200ms in an adhoc pip command that builds from source).

Reply:
Sure https://github.com/pypa/pip/compare/7cb7e1916cd25390644844a482bf0e1ee96e5c51..627851ac73d00fc68b2a6832f45691484e5814a9.